docs: Update Troubleshooting overview page (#13217)
openvinotoolkit/openvino: docs/install_guides/troubleshooting.md

@endsphinxdirective

> This guide provides general troubleshooting steps and solutions to possible issues that can be encountered while installing and configuration OpenVINO™.

Reviewer: Thanks! I missed that

> The [Issues & Solutions](./troubleshooting-issues.md) page lists common installation errors and their solutions. If you encountered a specific error while installing OpenVINO, check this page to see if there is a solution.

Reviewer: ```suggestion
The [Issues & Solutions](./troubleshooting-issues.md) page lists common installation and configuration errors, and their possible solutions. If you encountered a specific error while installing or configuring OpenVINO, check this page to see if there is a solution.
```

> The [Troubleshooting Steps](./troubleshooting-steps.md) page provides a set of instructions for diagnosing and resolving installation issues. If you had problems during installation, work through these steps to try and resolve your issue.

Reviewer 2: The guides are not just about installation but also "configuration".

Reviewer: I updated. :)

> The [Troubleshooting Steps](./troubleshooting-steps.md) page provides a set of instructions for diagnosing and resolving installation issues. If you had problems during installation, work through these steps to try and resolve your issue.

Reviewer: ```suggestion
The [Troubleshooting Steps](./troubleshooting-steps.md) page provides a set of instructions for diagnosing and resolving installation and configuration issues. If you had problems during installation and configuration, walk through these steps to try and resolve your issue.
```

> The [Troubleshooting Steps](./troubleshooting-steps.md) page provides a set of instructions for diagnosing and resolving installation issues. If you had problems during installation, please work through these steps to try and resolve your issue.

Reviewer: ```suggestion
The [Troubleshooting Steps](./troubleshooting-steps.md) page provides a set of instructions for diagnosing and resolving installation issues. If you had problems during installation, work through these steps to try and resolve your issue.
```
We decided to avoid the usage of "please" in technical writing. :D Just to be consistent.
